src/Avalonia.X11/Clipboard/ClipboardDataFormatHelper.cs

```diff
@@ -32,21 +33,39 @@ internal static class ClipboardDataFormatHelper
         if (atoms.GetAtomName(formatAtom) is { } atomName)
         {
             return atomName == MimeTypeTextUriList ?
-                DataFormat.File :
-                DataFormat.FromSystemName<byte[]>(atomName, AppPrefix);
+                DataFormat.File : DataFormat.FromSystemName<byte[]>(atomName, AppPrefix);
         }
 
         return null;
     }
 
-    public static IntPtr ToAtom(DataFormat format, IntPtr[] textFormatAtoms, X11Atoms atoms)
+    public static IntPtr ToAtom(DataFormat format, IntPtr[] textFormatAtoms, X11Atoms atoms, DataFormat[] dataFormats)
     {
         if (DataFormat.Text.Equals(format))
             return GetPreferredStringFormatAtom(textFormatAtoms, atoms);
 
         if (DataFormat.File.Equals(format))
             return atoms.GetAtom(MimeTypeTextUriList);
 
+        if (DataFormat.Bitmap.Equals(format))
+        {
+            DataFormat? pngFormat = null, jpegFormat = null;
+            foreach (var imageFormat in dataFormats)
+            {
+                if (imageFormat.Identifier is "image/png")
```

Comment:
Nit: introduce constants for mime types since we already have one for `text/uri-list` at the beginning of the file.